environment.yml

```diff
@@ -45,6 +45,9 @@ dependencies:
   - pip
   - pip:
     - pysuerga
+    - types-python-dateutil
+    - types-pytz
+    - types-setuptools
```

Comment:
These two are already at the end of the file, from conda-forge (not pip). I think we added them after you started this PR. Can you remove, and also check if we can add `types-python-dateutil` from conda-forge for consistency please.

Reply:
Indeed, they were added after I started working in the PR. I've made the changes you suggest.